test/parallel/test-runner-extraneous-async-activity.js

```diff
@@ -31,3 +31,17 @@ const { spawnSync } = require('child_process');
   assert.strictEqual(child.status, 1);
   assert.strictEqual(child.signal, null);
 }
+
+{
+  const child = spawnSync(process.execPath, [
+    '--test',
+    fixtures.path('test-runner', 'async-error-in-test-hook.mjs'),
+  ]);
+  const stdout = child.stdout.toString();
+  assert.match(stdout, /^# Warning: Hook "before" generated asynchronous activity after the test ended/m);
```

Comment:
Would this message scale well?
A full real-life project can have tens of hooks, and I am not sure this message would be helpful